Package.swift

```diff
@@ -1,9 +1,13 @@
-// swift-tools-version:5.3
+// swift-tools-version:5.5
 
 import PackageDescription
 
 let package = Package(
     name: "HTTP Client",
+    platforms: [
+        .iOS(.v15),
```

Comment:
Limiting iOS to >15 feels like a regression. Currently Jumpstart is compatible with iOS 14, and we's like to keep it that way in our app. How about adding "shims" for iOS <15 (see suggestion at the bottom of this [comment](https://github.com/jumpstart-pro/jumpstart-pro-ios/discussions/144#discussioncomment-3069592))?

Reply:
Great call. I added a backfill for iOS 14 and macOS 10/11.